src/NServiceBus.Core.Tests/Sagas/ComponentTests/When_completing_a_saga_with_no_defined_correlation_property.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_correlation_property : SagaPersisterTests<SagaWithoutCorrelationProperty, SagaWithoutCorrelationPropertyData>
```

Comment:
So far we have your opinion @andreasohlund When @Scooletz @tmasternak @SeanFeldman @yvesgoeleven and I worked on writing the component tests that were adapted from the inmem tests which already use similar naming we went through a series of renaming and cleanups that lead to the state that the code is now. I'm not saying you have no point. I wanted to highlight that there was significant amount of work, thoughts and alignment going on already and now if we change this here we have to also backport it to SF Persistence to align and essentially consult the initial TF. Is it worth it?